data/osfamily/common.yaml

```diff
@@ -0,0 +1,2 @@
+---
+postfix::params::master_os_template: "postfix/master.cf.default.erb"
```

Comment:
Wouldn't this be better placed in the existing `data/common.yaml`?

Reply:
What do you mean by "better"?  
This Parameter is clearly related to `osfamily`.  So IMHO this is the most obvious Location: parallel to the other Locations of `postfix::params::master_os_template`.

Reply:
I moved the Hiera-Parameter to `data/common.yaml` now.  
So forget all of the Above.